packages/flame/lib/src/effects/sequence_effect.dart

```diff
@@ -124,9 +173,21 @@ class _SequenceEffectEffectController extends EffectController {
     for (final effect in effects) {
       totalDuration += effect.controller.duration ?? 0;
     }
+
+    // Abort early
+    if (totalDuration == 0.0) {
+      return totalDuration;
+    }
+
     if (alternate) {
       totalDuration *= 2;
+
+      if (alternatePattern == AlternatePattern.doNotRepeatLast) {
+        totalDuration -= effects.first.controller.duration ?? 0;
```

Comment:
How come the first one is removed here too? :thinking:

Reply:
Hi. This has been explained in the PR, code comments, and here:
https://github.com/flame-engine/flame/pull/3322#discussion_r1828604667